stream/BidirectionalStream.java

```diff
@@ -76,7 +76,7 @@ public BidirectionalStream(TypeDBStub stub, RequestTransmitter transmitter) {
         UUID requestID = UUID.randomUUID();
         ResponseCollector.Queue<ResPart> collector = resPartCollector.queue(requestID);
         dispatcher.dispatch(request.setReqId(UUIDAsByteString(requestID)).build());
-        ResponsePartIterator iterator = new ResponsePartIterator(requestID, collector, this);
+        ResponsePartIterator iterator = new ResponsePartIterator(requestID, collector, dispatcher);
```

Comment:
we no longer rely on the bidirectional stream to retrieve the error, so we can revert this to only pass in the dispatcher rather than the entire stream